Stratis.Bitcoin/Wallet/IWalletManager.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using System.Collections.Generic;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Wallet
+{
+    /// <summary>
+    /// Interface for a manager providing operations on wallets.
+    /// </summary>
+    public interface IWalletManager : IDisposable
+    {
+		/// <summary>
+		/// List all spendable transactions from all accounts
+		/// </summary>
+		/// <returns>A collection of spendable outputs</returns>
+		List<UnspentInfo> GetSpendableTransactions();
```

Comment:
This returns all the txs for all the wallets. is that the intention?
Also it should return an IEnumerable.